src/cclib/parser/orcaparser.py

```diff
@@ -551,6 +551,55 @@ def extract(self, inputfile, line):
                 self.gbasis.append(gbasis_tmp[bas_atname])
             del self.tmp_atnames
 
+        """
+        --------------------------
+        THERMOCHEMISTRY AT 298.15K
+        --------------------------
+        """
+        if 'THERMOCHEMISTRY AT' == line[:18]:
+            conv = lambda x: utils.convertor(float(x), "hartree", "eV")
+
+            next(inputfile)
+            next(inputfile)
+            self.temperature = float(next(inputfile).split()[2])
```

Comment:
We don't do this kind of alignment. Could you use one space at each side of all binary operators?